From 319f460edc9eaff632fcc47f9efa037f7a16c750 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Fri, 4 Mar 2016 18:18:37 +0800
Subject: [PATCH 1605/1691] MLK-12478-2 dts: imx7d-lpddr3-arm2: add lpsr mode
 state for flexcan pins

commit 319f460edc9eaff632fcc47f9efa037f7a16c750 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

add lpsr mode state for flexcan pins

Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
(cherry picked from commit 535699f47fbd7fb22a435ca2047560ee20687392)
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index 00c7113..b5765a2 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -228,15 +228,17 @@
 };
 
 &flexcan1 {
-	pinctrl-names = "default";
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_flexcan1>;
+	pinctrl-1 = <&pinctrl_flexcan1>;
 	xceiver-supply = <&reg_can1_3v3>;
 	status = "disabled";
 };
 
 &flexcan2 {
-	pinctrl-names = "default";
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_flexcan2>;
+	pinctrl-1 = <&pinctrl_flexcan2>;
 	xceiver-supply = <&reg_can2_3v3>;
 	status = "disabled";
 };
-- 
1.9.1

